subtests/docker_cli/run_volumes_selinux/run_volumes_selinux.py

```diff
@@ -72,27 +75,33 @@ def init_container(self, volume=None, volumes_from=None):
         self.sub_stuff['fds'].append(write_fd)
         self.sub_stuff['fds'].append(read_fd)
         dkrcmd = dockercmd.AsyncDockerCmd(self, 'run', subargs)
-        # TODO: Fix use of dkrcmd.stdin when appropriate class mech. available
         dkrcmd.execute(read_fd)
         dkrcmd.stdin = write_fd
         os.close(read_fd)  # no longer needed
-        os.write(write_fd, 'echo "Started"\n')
-        self.failif(not wait_for_output(lambda: dkrcmd.stdout,
-                                        'Started',
-                                        timeout=20),
-                    "Error starting container %s: %s" % (name, dkrcmd))
+
+        # This could take a while if image needs pulling
+        find_name = lambda: str(dc.list_containers_with_name(name)) != []
+        self.failif_ne(wait_for_output(find_name,
```

Comment:
Nope, the confusion is all mine.  I intended to use your suggestion of ``utils.wait_for()`` but a bird suddenly flew past the window.  Out of the corner of my eye, I swore it had my toothbrush in it's beak.  Unfortunately, between getting up to check, and sitting back down, my cat attacked me.  Then while getting a new pair of pants, I found the pack of dental-floss that went missing last week.   Unfortunately, justice had already (mistakenly) been delivered to that particular clepto-bird.  RIP.